SimFastTiming/FastTimingCommon/src/BTLElectronicsSim.cc

```diff
@@ -94,3 +183,19 @@ void BTLElectronicsSim::updateOutput(BTLDigiCollection &coll,
     coll.push_back(dataFrame);    
   }
 }
+
+float BTLElectronicsSim::sigma2_pe(const float& Q, const float& R) const {
+  
+  float OneOverR = 1./R;
+  // --- This is Eq. (17) from Nucl. Instr. Meth. A 564 (2006) 185
+  float sigma2 = Q*OneOverR*OneOverR + 
```

Comment:
precalculate `OneOverR*OneOverR` and replace it wherever possible in this formula
(can even factor it out of all the terms)

Reply:
Done